drivers/hmis/app/graphql/types/hmis_schema/omnisearch_result.rb

```diff
@@ -17,7 +17,7 @@ def self.resolve_type(object, _context)
       elsif object.is_a?(Hmis::Hud::Client)
         Types::HmisSchema::Client
       else
-        raise "#{object.class.name} is not a valid omnisearch result"
+        raise "#{object.class.sti_name} is not a valid omnisearch result"
```

Comment:
Might make more sense to leave this as name, since it's displayed to the developer, not using it to match something in the db.